src/controller/jobservice/schedule.go

```diff
@@ -0,0 +1,68 @@
+//  Copyright Project Harbor Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package jobservice
+
+import (
+	"context"
+	"github.com/goharbor/harbor/src/lib/errors"
+	"github.com/goharbor/harbor/src/lib/q"
+	"github.com/goharbor/harbor/src/pkg/scheduler"
+)
+
+var (
+	// SchedulerCtl ...
+	SchedulerCtl = NewSchedulerCtrl()
+)
+
+// SchedulerController interface to manage schedule
+type SchedulerController interface {
```

Comment:
Get(), Create() and Delete() are more clean?

Reply:
Updated